scripts/mysql_system_users.sql

```diff
@@ -91,12 +94,23 @@ GRANT BACKUP_ADMIN, LOCK TABLES, PROCESS, RELOAD, REPLICATION CLIENT, SUPER, SHU
 CREATE ROLE 'mysql.pxc.sst.role'@localhost;
 REVOKE ALL PRIVILEGES, GRANT OPTION FROM 'mysql.pxc.sst.role'@localhost;
 
-GRANT BACKUP_ADMIN, LOCK TABLES, PROCESS, RELOAD, REPLICATION CLIENT, SUPER, CLONE_ADMIN,GROUP_REPLICATION_STREAM,SYSTEM_USER, SHUTDOWN,CONNECTION_ADMIN, CREATE USER ON *.* TO 'mysql.pxc.sst.role'@localhost WITH GRANT OPTION;
-GRANT INSERT,DELETE ON mysql.plugin TO 'mysql.pxc.sst.role'@localhost WITH GRANT OPTION;
-GRANT CLONE_ADMIN, SYSTEM_VARIABLES_ADMIN, SUPER, SHUTDOWN, EXECUTE ON *.* to 'mysql.pxc.sst.role'@localhost WITH GRANT OPTION;
-GRANT SELECT,UPDATE,INSERT ON performance_schema.* TO 'mysql.pxc.sst.role'@localhost WITH GRANT OPTION;
-
+-- Needed by sst_xtrabackup-v2
+GRANT LOCK TABLES, PROCESS, RELOAD, REPLICATION CLIENT, SUPER ON *.* TO 'mysql.pxc.sst.role'@localhost;
 GRANT ALTER, CREATE, SELECT, INSERT ON PERCONA_SCHEMA.xtrabackup_history TO 'mysql.pxc.sst.role'@localhost;
+
+-- Common. WITH GRANT OPTION is needed by clone-sst script
+GRANT BACKUP_ADMIN, EXECUTE ON *.* TO 'mysql.pxc.sst.role'@localhost WITH GRANT OPTION;
+GRANT SELECT ON performance_schema.* TO 'mysql.pxc.sst.role'@localhost WITH GRANT OPTION;
```

Comment:
Do we need the GRANT command in line number 115, when we are doing the same here, but with `GRANT OPTION`?
```
114: -- For some reason this is also needed, although the docs say BACKUP_ADMIN is enough
115: GRANT SELECT ON performance_schema.* TO 'mysql.pxc.sst.role'@localhost;
```

Reply:
Good point. Fixed